google-cloud-sdk/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_name=google-cloud-sdk
+pkg_description="Google Cloud SDK is a set of tools that you can use to manage resources and applications hosted on Google Cloud Platform."
+pkg_upstream="https://cloud.google.com/sdk/"
+pkg_origin=core
+pkg_version="231.0.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_source="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/${pkg_name}-${pkg_version}-linux-x86_64.tar.gz"
+pkg_shasum="7f1a5ca6e521fb3b4a17c557d0107a032f3f486b7071ee4895ded1077ffda49e"
+pkg_deps=(core/python2)
+pkg_bin_dirs=(bin)
+
+do_build() {
+    return 0
+}
+
+do_install() {
+    bash $HAB_CACHE_SRC_PATH/$pkg_name/install.sh --quiet
+    mkdir -p $pkg_prefix/lib
+    cp -r $HAB_CACHE_SRC_PATH/$pkg_name/lib $pkg_prefix/
+    mkdir -p $pkg_prefix/bin
+    cp -r $HAB_CACHE_SRC_PATH/$pkg_name/bin/* $pkg_prefix/bin/
+    hab pkg install core/python2 -bf
```

Comment:
I'd love some feedback on a better approach here. I need python to be in the runtime env or the gcloud sdk tools error with `/bin/gcloud: line 129: exec: python: not found`. Even though I've declared `core/python2` as a `pkg_dep`, it seems like it only adds the python binpath to the PATH env during build time. I also tried adding the python binpath via `push_runtime_env PATH $(pkg_path_for core/python2)/bin` but I got:
```   google-cloud-sdk: PUSH TO RUN
   google-cloud-sdk: ERROR: Cannot directly manipulate environment variable PATH! Add appropriate entries to the 'pkg_deps' variable in plan.sh instead!
   google-cloud-sdk: Build time: 0m2s
   google-cloud-sdk: Exiting on error```